### PR TITLE
Beets 323 buttons

### PR DESCRIPTION
**Pull request by jwehrman25**
### Definition of Done:

* [x] Have you checked your [Jira](https://bee-cs458.atlassian.net/jira/software/projects/BEETS/boards/2/backlog) task, to make sure the user acceptance criteria are met?
* [x] Does your code have comments describing its function and reason?

### Tests

* [ ] Do you have Jest tests?
* [ ] Have you documented your testing criteria and edge cases in your Jira task?

### Source Control

* [x] Is all relevant code checked-in?
* [x] Is this branch ready to be reviewed by the Tech Lead?

### Formatting

* [x] Do your variable names make sense (to others)?
* [x] Have you removed unnecessary Console.logs? 
* [ ] Is your code style consistent with other code in the file or the [Style Guide](https://google.github.io/styleguide/jsguide.html)

### CODE REVIEW

File: client/src/components/UserTable/UserTable.js
Comment (levertyk) [inline client/src/components/UserTable/UserTable.js:1]: There appears to be a typo on this change

File: client/src/components/UserTable/UserTable.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ export default function UsersTable(props) {
             <Modal.Footer>
               <Button
                 variant="primary"
+                className="`beets_buttons"
```
Comment (levertyk) [inline client/src/components/UserTable/UserTable.js:100]: There appears to be a typo on this line.

**Review by levertyk — CHANGES_REQUESTED** [review]
There is a typo in UserTable. Please fix before merge.

**Review by levertyk — APPROVED** [review]
Perfect
